scripts/windows/test_task.bat

```diff
@@ -115,6 +115,7 @@ hMailServer.exe /verysilent
 cd %APPVEYOR_BUILD_FOLDER%
 %PHP_BUILD_DIR%\php.exe -dextension_dir=%PHP_BUILD_DIR% -dextension=com_dotnet appveyor\setup_hmailserver.php
 
+if EXIST %PHP_BUILD_DIR%\test_file_cache rmdir /s /q %PHP_BUILD_DIR%\test_file_cache
```

Comment:
In this case instead of `mkdir -p` (ensure exists); I opted to clean the folder to ensure leftover state doesn't affect anything, or confuse anyone. I am unsure if this is correct.